byterun/pyvm2.py

```diff
@@ -899,11 +999,21 @@ def byte_MAKE_FUNCTION(self, argc):
         if PY3:
             name = self.pop()
         else:
+            # Pushes a new function object on the stack. TOS is the code
+            # associated with the function. The function object is defined to
+            # have argc default parameters, which are found below TOS.
             name = None
         code = self.pop()
-        defaults = self.popn(argc)
         globs = self.frame.f_globals
-        fn = Function(name, code, globs, defaults, None, self)
+        if PY3 and sys.version_info.minor >= 6:
+            closure = self.pop() if (argc & 0x8) else None
```

Comment:
is there a reason to use hex-literals for values under 10?

Reply:
@llllllllll I took the comparisons from the [Python bytecode documentation](https://docs.python.org/3.6/library/dis.html) directly.